src/realm/impl/transact_log.hpp

```diff
@@ -1414,6 +1447,14 @@ class TransactReverser {
         return true;
     }
 
+    bool typed_link_initialize(ColKey col, TableKey dest)
+    {
+        // FIXME: this is not the reverse, but does it matter?
```

Comment:
Maybe name it `TypedLinkChange` and say that it represents a change in what backlink columns exist due to typed link things rather than the more specific meaning it has now? No functional change but it'd make it so that the operation being its own inverse is correct.

Reply:
Great idea.